### ISSUE TITLE
Works fine under 10.12!

### ISSUE THREAD

**Issue by Daij-Djan** [CamHenlin/imessageclient#23]
Works fine under 10.12! :) Maybe you wanna update the readme? :)

**Comment by rafe-s** [comment]
Yes, It'd be great if this was better maintained. For example, the Address Book functionality isn't fully implemented and I keep getting warning Accessibility warning when running it over SSH even though I've authorized sshd_keygen_wrapper.
